### ISSUE TITLE
Handle qrel binarization

### ISSUE THREAD

**Issue by JMMackenzie** [rankbiased/rbstar#7]
We should check how TREC handle qrel binarization and follow the same process, paying attention to types and trying to make things as clear as possible.

**Comment by mpetri** [comment]
but RBP supports float gain values. do we need to do this?

**Comment by mpetri** [comment]
actually right now it is a bit of a bug that we cast the value to int and if the max value is > 1 we don't normalize the metric to [0,1] (I guess the RBP paper also doesn't do this)

**Comment by JMMackenzie** [comment]
Yeah, we should check how to handle - I believe we should allow gain mapping to floats like you suggested, but we'd need to be explicit about it somehow. Perhaps just let the qrels file handle floats?
